Source/WebCore/page/Page.cpp

```diff
@@ -1793,6 +1793,8 @@ void Page::doAfterUpdateRendering()
 
     DebugPageOverlays::doAfterUpdateRendering(*this);
 
+    m_renderingUpdateRemainingSteps.last().remove(RenderingUpdateStep::PrepareCanvasesForDisplay);
+
```

Comment:
Indentation looks weird

Reply:
This is a GH diff viewer issue. It's correct in the patch.

<img width="455" alt="image" src="https://user-images.githubusercontent.com/156077/170385319-7b76abc7-5281-42b0-b131-328eb3a9d759.png">